src/components/JobWizard/JobWizardSteps/JobWizardDataInputs/JobWizardDataInputs.js

```diff
@@ -35,6 +35,7 @@ const JobWizardDataInputs = ({ formState, stepIsActive }) => {
           exitEditModeTriggerItem={stepIsActive}
           fieldsPath={`${DATA_INPUTS_STEP}.dataInputsTable`}
           formState={formState}
+          id="data-inputs-form-table"
```

Comment:
There is no sense to pass this id here. It should be unique for the place where it's used. Or just add this id inside the component.